commons/src/main/java/org/infinispan/commons/dataconversion/BinaryEncoder.java

```diff
@@ -53,7 +53,7 @@ private static boolean isTypeExcluded(Class<?> type) {
    }
 
    private boolean skipEncoding(Object source) {
-      return isTypeExcluded(source.getClass());
+      return source == null || isTypeExcluded(source.getClass());
```

Comment:
Why @anistor? ```null``` can be written to the cache value, and can also be read from certain commands (e.g. return of put), where should they be handed?